frontend/ui/widget/button.lua

```diff
@@ -108,7 +108,7 @@ function Button:init()
                     self.label_widget = TextBoxWidget:new{
                         text = self.text,
                         line_height = 0,
-                        alignment = "center",
+                        alignment = self.align,
```

Comment:
This is fixing a bug right ? (and not you tweaking its behaviour for your need).
(left-aligned multi line buttons ended up centered I guess, don't remember where we have left aligned buttons.)

Reply:
I believe it's a bug, and also me tweaking its behavior for my need. But when a single-line button is left-aligned, it should keep being so when multi-lined?

Reply:
Yes, that's why I also think it's a bug.